DigitalOffice/TimeManagementService/Database/Entities/TypeLeave.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace TimeManagementService.Database.Entities
+{
+    public enum TypeLeave
```

Comment:
Do we need this as a part of database design? Why not just using int? Enum will be later described, implementing some logic, I guess

Reply:
deleted

Reply:
I don't think that such a hurry is necessary in these kinds of changes.

When you ask these questions, @zzzloba you should propose your own ideas, not just criticize the author; or if you're just curious, then point it out from the beginning. At least it makes me confused every time.

As for me on the enum thing, I didn't get what the consensus is here. If we don't use enum, how will we implement such a model | a table | etc. ?
Ok, if we change the property in Leave to int TypeLeave, then it should be
public int TypeLeaveId
then we create a new entity called TypeLeaves (and by the way I think we're mistaken in this kind of naming. In english it's called Leave Type, not Type Leave) that contains properties like:
Id (TypeLeaveId), TypeName, Description, etc.
And only then that change to int will work.

If enum implementation is considered on the business logic layer, then it's a bad practice: we should be able to make changes to the database and not the whole lot of code that we've written. Imagine if we decipher each number to specific TypeLeave, for example, "Vacation". 
And later on, Vacation is responsible for number 3 and not 2 as it was previously. Can you now imagine the amount of extra work? :)

Reply:
> I don't think that such a hurry is necessary in these kinds of changes.
> 
> When you ask these questions, @zzzloba you should propose your own ideas, not just criticize the author; or if you're just curious, then point it out from the beginning. At least it makes me confused every time.
> 
> As for me on the enum thing, I didn't get what the consensus is here. If we don't use enum, how will we implement such a model | a table | etc. ?
> Ok, if we change the property in Leave to int TypeLeave, then it should be
> public int TypeLeaveId
> then we create a new entity called TypeLeaves (and by the way I think we're mistaken in this kind of naming. In english it's called Leave Type, not Type Leave) that contains properties like:
> Id (TypeLeaveId), TypeName, Description, etc.
> And only then that change to int will work.
> 
> If enum implementation is considered on the business logic layer, then it's a bad practice: we should be able to make changes to the database and not the whole lot of code that we've written. Imagine if we decipher each number to specific TypeLeave, for example, "Vacation".
> And later on, Vacation is responsible for number 3 and not 2 as it was previously. Can you now imagine the amount of extra work? :)

Mb just move it to another folder, not "Database/Entities" ;)

Reply:
Maybe folder "utils"?

Reply:
How will we add to the enum new type leave,  if we will want expand list TypeLeave, example: add new row "Hackathon".